go_modules/spec/dependabot/go_modules/file_parser_spec.rb

```diff
@@ -47,7 +47,7 @@
         parser.parse.select(&:top_level?)
       end
 
-      its(:length) { is_expected.to eq(3) }
+      its(:length) { is_expected.to eq(2) }
```

Comment:
The fixture file is located at https://github.com/dependabot/dependabot-core/blob/main/go_modules/spec/fixtures/go_mods/go.mod

There are three direct dependencies in the fixture, but rsc.io/qr uses a replace directive, so it is not counted as a top-level anymore with this change.